deployment/terraform/assets/redis_dashboard_data.json

```diff
@@ -1,25 +1,32 @@
 {
   "__inputs": [
     {
-      "name": "DS_PROMETHEUS",
-      "label": "Prometheus",
+      "name": "DS_LOADTEST-SOURCE",
+      "label": "loadtest-source",
       "description": "",
       "type": "datasource",
       "pluginId": "prometheus",
       "pluginName": "Prometheus"
     }
   ],
+  "__elements": {},
   "__requires": [
+    {
+      "type": "panel",
+      "id": "gauge",
+      "name": "Gauge",
+      "version": ""
+    },
     {
       "type": "grafana",
       "id": "grafana",
       "name": "Grafana",
-      "version": "6.5.2"
+      "version": "10.2.3"
     },
     {
       "type": "panel",
       "id": "graph",
-      "name": "Graph",
+      "name": "Graph (old)",
```

Comment:
Is this expected?

Reply:
Yep, it is. The new panel is called "timeseries", as you can see below the "type" of panel has changed. And the old one is called "graph".